packages/runtime/src/webcontainer/on-demand-boot.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Lightweight wrapper around WebContainer.boot
```

Comment:
```suggestion
 * Lightweight wrapper around `WebContainer.boot`
```

Reply:
This comment is a bit hard to read and could benefit from a slight rewording.
```
Lightweight wrapper around "WebContainer.boot" to only boot if requested on iOS, since booting might risk running out of memory.
```